lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerFactory.java

```diff
@@ -35,13 +37,16 @@
  */
 public final class DefaultRemoteManagerFactory implements RemoteManagerFactory {
```

Comment:
We should now be able to make this package private, as other code should be coded against `RemoteManagerFactory` anyhow, right?